src/lib/utils.ts

```diff
@@ -50,44 +42,9 @@ export function parseDkimTagList(dkimValue: string): Record<string, string> {
 		}
 		const key = part.slice(0, i).trim();
 		const value = part.slice(i + 1).trim();
-		switch (key) {
-			case "v":
-				if(value !== 'DKIM1') {
```

Comment:
this should say DKIM1 or 1, and by default should continue since this value isn't constrained by [RFC 6376](https://datatracker.ietf.org/doc/html/rfc6376).